docs/configuration.md

```diff
@@ -1007,6 +1007,28 @@ Apart from these, the following properties are also available, and may be useful
   </td>
   <td>3.3.0</td>
 </tr>
+<tr>
+  <td><code>spark.shuffle.service.db.enabled</code></td>
+  <td>true</td>
+  <td>
+    Store External Shuffle service state on local disk so that when the external shuffle service is restarted, it will
+    automatically reload info on current executors.  This only affects standalone mode (yarn always has this behavior
+    enabled).  You should also enable <code>spark.worker.cleanup.enabled</code>, to ensure that the state
+    eventually gets cleaned up.  This config may be removed in the future.
+  </td>
+  <td>3.0.0</td>
+</tr>
```

Comment:
This should be in the section at the bottom - in `External Shuffle service(server) side configuration options` section.

Having said that, I was wrong about `spark.shuffle.service.db.enabled` - it is always enabled in yarn mode - so we cannot control it with `.enabled` flag.
The newly introduced `spark.shuffle.service.db.backend` is relevant though; but can we add a blurb that it is relevant for yarn and standalone - with db enabled by default for yarn, and to look at standalone.md for more on details there ?

Currently, we have the yarn shuffle service config in `configuration.md`; so can we

Reply:
Does [e124c84](https://github.com/apache/spark/pull/37610/commits/e124c845eee9c534dff913baa25c3b5a65c83113)  look better?

Reply:
[4d5f44c](https://github.com/apache/spark/pull/37610/commits/4d5f44ce0b924e8f52f787843ad4759a1731dc4a) move the to bottom of this section

Reply:
https://spark.apache.org/docs/latest/configuration.html#shuffle-behavior

Is this table item should sorted by `Property Name`? Seems not follow this rule.

Reply:
Actually, that section I mentioned is for push based shuffle - so probably does not make sense for our changes here.

Reply:
Move it back to after https://github.com/apache/spark/blob/6e62b93f3d1ef7e2d6be0a3bb729ab9b2d55a36d/docs/configuration.md?plain=1#L1000-L1008
?

They have been add into `spark-standalone.md` and `running-on-yarn.md`.Is it necessary to keep them in `configuration.md`?

Reply:
You are right, we need it only in standalone and yarn docs for now - not in configuration

Reply:
removed